### DISCUSSION TITLE
Tabby can not show the symbol for neovim status tool

### DISCUSSION THREAD

**Discussion by qinf** [Eugeny/tabby#5960] [General]
When i use tabby, it can not show the symbol for the status tool as follow:
<img width="1421" alt="image" src="https://user-images.githubusercontent.com/1277293/159104081-a5df66bc-b651-4819-936b-5ec165aa7713.png">
But `Wezterm` can show it like this:
![image](https://user-images.githubusercontent.com/1277293/159104104-ddef1a28-1dfc-441e-9d8b-b3607526f4c2.png)

Am I use the wrong configuration? Thank you for your help.

**Comment by Eugeny** [comment]
Which version are you on?
* This means that your current font doesn't include these characters. Usually you'd need to use a special patched version of a font, e.g. one from https://www.nerdfonts.com/
* However starting with v1.0.173, a fallback font is included with Tabby which should include all of these characters.

**Reply by qinf** [reply]
I use version `1.0.170`, and when I found a patched verion of a font, it show the characters correctly.
<img width="1424" alt="image" src="https://user-images.githubusercontent.com/1277293/159155793-e16b23c7-4919-48bb-bdde-762965ea7605.png">
But when i updated Tabby to `1.0.173`, it still have the same problem as follow:
<img width="1416" alt="image" src="https://user-images.githubusercontent.com/1277293/159155765-9a824802-5ddd-499a-bf4f-c09c1f92d66f.png">

**Reply by Eugeny** [reply]
Fixed in the nightly build
